### ISSUE TITLE
Is the fontawesome.io website open sourced?

### ISSUE THREAD

**Issue by Bilal-io** [FortAwesome/Font-Awesome#10441]
I want to suggest a UX advice regarding the error message appearing when you search for an icon and it doesn't exist, however I cannot tell if the website is open sourced so I can contribute to it. 

The error message is red with too much text. In my opinion the text should be minimized and the text color to be changed to gray instead. Gray refers to emptiness where red is for warnings and alerts, this would arouse a user and it is not necessary. 

Thank you

**Comment by tagliala** [comment]
Yes, documentation is open sourced: https://github.com/FortAwesome/Font-Awesome/tree/master/src

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
